### PR TITLE
Add failNow overload with string argument

### PR DESCRIPTION
**Pull request by ndr-brt**
As motivated in #87

### CODE REVIEW

**Comment by jponge** [comment]
How about using `VertxException` rather than `NoStackTraceThrowable`? I think it's useful to have the stack trace anyway.

**Comment by ndr-brt** [comment]
@jponge yes, but not in this case I think. 
The `failNow` is used at the end of the test, the test fails but not because an exception, so the stacktrace won't give useful informations.

**Comment by jponge** [comment]
Right

File: src/test/java/io/vertx/junit5/VertxTestContextTest.java [hunk 1]
```diff
@@ -456,4 +457,15 @@ void check_unsatisifed_checkpoint_callsites() {
...
+
+  @Test
+  @DisplayName("Check failNow() called with a string")
+  void check_fail_now_called_with_a_string() {
+    VertxTestContext context = new VertxTestContext();
+
+    context.failNow("error message");
+
+    assertThat(context.failed()).isTrue();
+    assertThat(context.causeOfFailure()).isInstanceOf(NoStackTraceThrowable.class);
```
Comment (jponge) [inline src/test/java/io/vertx/junit5/VertxTestContextTest.java:469]: You should also chain `hasMessage` to verify the exception message 😉

**Review by jponge — CHANGES_REQUESTED** [review]

File: src/test/java/io/vertx/junit5/VertxTestContextTest.java (hunk 1 above)
Comment (ndr-brt) [reply src/test/java/io/vertx/junit5/VertxTestContextTest.java:469]: right.
done :)

**Review by jponge — APPROVED** [review]
Thank you!

**Pull request merged**